plugins/hydra_ax_sweeper/example/conf/config.yaml

```diff
@@ -0,0 +1,32 @@
+defaults:
+  - hydra/sweeper: ax
+
+# Banana function
+# https://en.wikipedia.org/wiki/Rosenbrock_function
+banana:
+  # To be minimized
+  x: ???
+  y: ???
+  
+
+hydra:
+  ax:
+    experiment:
+      # Banana function should be minimized (Ax defaults to maximize)
+      minimize: true
+
+    early_stop:
+      # Number of epochs without a significant improvement from
+      # the currently known best parameters
+      # An Epoch is defined as a batch of trials executed in parallel
+      max_epochs_without_improvement: 100
```

Comment:
this seems like a very defensive default value.

Reply:
Changing it to 10